lib/livebook_web/router.ex

```diff
@@ -34,6 +34,8 @@ defmodule LivebookWeb.Router do
     live "/sessions/:id/settings/runtime", SessionLive, :runtime_settings
     live "/sessions/:id/settings/file", SessionLive, :file_settings
     live "/sessions/:id/bin", SessionLive, :bin
+    live "/sessions/:id/export", SessionLive, :export
+    get "/sessions/:id/export/download", SessionController, :download_source
```

Comment:
We will likely have different URLs per export. But it is fine, we can namespace them later.

Reply:
Definitely, maybe format as URL parameter.